components/brave_wallet_ui/components/desktop/wallet-onboarding/recovery/index.tsx

```diff
@@ -50,7 +52,10 @@ function OnboardingBackup (props: Props) {
         )}
       </RecoveryPhraseContainer>
       <Tooltip text={getLocale('braveWalletToolTipCopyToClipboard')}>
-        <CopyButton onClick={onCopy}>{getLocale('braveWalletButtonCopy')}</CopyButton>
+        <CopyButton onClick={onCopy}>
+          {recoverPhraseCopied && <BigCheckMark />}
+          {recoverPhraseCopied ? 'Copied' : getLocale('braveWalletButtonCopy')}
```

Comment:
String needs to be added here `getLocale('braveWalletButtonCopied')` 😎

Reply:
thank you, changing 👌